phabfive/constants.py

```diff
@@ -30,3 +30,13 @@
 TICKET_PRIORITY_NORMAL = "normal"
 TICKET_PRIORITY_LOW = "low"
 TICKET_PRIORITY_WISH = "wish"
+
+#dict for all default(?) priority type
```

Comment:
```suggestion
# dict for all default(?) priority type
```

This comment line don't make sense, is it a question for us or something else? In general if this is a question between a list or dict, i think it mostly depends on what type of application it is used for. In the long term we are going to regular Enum:s in python so it is not critical that we sort this out now. If a list works, we can settle for it for now and we update it in the future. But if the comments is not needed, please remove it

Reply:
I had to admit I was lazy. On one hand I wanted to atleast to have the chance to assign priorities but on another hand I am stuck with my work that I just hardcoded someting instead to do a proper implementation of using /api/maniphest.priority.search to extract the priority that different Phabricator instances setup.... thus the (?) mark - maybe I would put a FIXME: instead?

Reply:
and I have to say, been 10+ years before I code again (was Java -> picked up python because I needed something at work to extract data from my company's Phrabricator instance) - appreciate your effort in reviewing my code + guilty as charged.